tools/util.py

```diff
@@ -359,11 +359,17 @@ def extract_number(pattern, string):
 
 
 def parse_wrk_output(output):
-    req_per_sec = None
+    stats = {}
+    stats['req_per_sec'] = None
+    stats['max_latency'] = None
     for line in output.split("\n"):
-        if req_per_sec is None:
-            req_per_sec = extract_number(r'Requests/sec:\s+(\d+)', line)
-    return req_per_sec
+        if stats['req_per_sec'] is None:
+            stats['req_per_sec'] = extract_number(r'Requests/sec:\s+(\d+)',
+                                                  line)
+        if stats['max_latency'] is None:
+            stats['max_latency'] = extract_number(
+                r'Latency(?:\s+(\d+).\d+[a-z]+){3}', line)
```

Comment:
Note that the latency can be expressed with different units: "s" (seconds), "ms" (milliseconds), "us" (microseconds)
Before returning the value, you should figure out the unit, and convert to milliseconds.